app/src/androidTest/java/ch/hikemate/app/ui/auth/CreateAccountScreenTest.kt

```diff
@@ -0,0 +1,113 @@
+package ch.hikemate.app.ui.auth
+
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.performClick
+import androidx.compose.ui.test.performTextInput
+import ch.hikemate.app.model.authentication.AuthRepository
+import ch.hikemate.app.model.authentication.AuthViewModel
+import ch.hikemate.app.ui.components.BackButton
+import ch.hikemate.app.ui.navigation.NavigationActions
+import com.kaspersky.kaspresso.testcases.api.testcase.TestCase
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.times
+import org.mockito.kotlin.any
+import org.mockito.kotlin.eq
+import org.mockito.kotlin.verify
+
+class CreateAccountScreenTest : TestCase() {
+  @get:Rule val composeTestRule = createComposeRule()
+
+  private lateinit var navigationActions: NavigationActions
+  private lateinit var authRepository: AuthRepository
+  private lateinit var authViewModel: AuthViewModel
+
+  private val TEST_NAME = "Test Name"
+  private val TEST_EMAIL = "test@example.com"
+  private val TEST_PASSWORD = "password"
+  private val TEST_DIFFERENT_PASSWORD = "different_password"
+
+  @Before
+  fun setUp() {
+    navigationActions = mock(NavigationActions::class.java)
+    authRepository = mock(AuthRepository::class.java)
+    authViewModel = AuthViewModel(authRepository)
+
+    composeTestRule.setContent {
+      CreateAccountScreen(navigationActions = navigationActions, authViewModel = authViewModel)
+    }
+  }
+
+  @Test
+  fun everythingIsOnScreen() {
+    composeTestRule.onNodeWithTag(BackButton.BACK_BUTTON_TEST_TAG).assertIsDisplayed()
+    composeTestRule.onNodeWithTag(CreateAccountScreen.TEST_TAG_TITLE).assertIsDisplayed()
+    composeTestRule.onNodeWithTag(CreateAccountScreen.TEST_TAG_NAME_INPUT).assertIsDisplayed()
+    composeTestRule.onNodeWithTag(CreateAccountScreen.TEST_TAG_EMAIL_INPUT).assertIsDisplayed()
+    composeTestRule.onNodeWithTag(CreateAccountScreen.TEST_TAG_PASSWORD_INPUT).assertIsDisplayed()
+    composeTestRule
+        .onNodeWithTag(CreateAccountScreen.TEST_TAG_CONFIRM_PASSWORD_INPUT)
+        .assertIsDisplayed()
+    composeTestRule.onNodeWithTag(CreateAccountScreen.TEST_TAG_SIGN_UP_BUTTON).assertIsDisplayed()
```

Comment:
Nitpick: You assigned a test tag to the Column (`Screen.CREATE_ACCOUNT`). Why isn't it checked here?

Reply:
This is mostly used for navigation (has to have a global tag for a whole screen). Do you think we should still check that the screen itself is displayed ?

Reply:
If it's checked in another test, lgtm, since other tests tags implies it is displayed too